From 5d934d94573575a531d14b5309f4e1c901f3b5f2 Mon Sep 17 00:00:00 2001
From: Yu Kuai <yukuai3@huawei.com>
Date: Mon, 19 Dec 2022 17:57:01 +0800
Subject: [PATCH] block: always align request_wrapper to cacheline
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 5d934d94573575a531d14b5309f4e1c901f3b5f2
Modified-by-SEL: No


hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I65K8D
CVE: NA

--------------------------------

____cacheline_aligned_in_smp will only be effective if CONFIG_SMP is
enabled, use ____cacheline_aligned instead.

Signed-off-by: Yu Kuai <yukuai3@huawei.com>
Reviewed-by: Hou Tao <houtao1@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 block/blk-mq.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/blk-mq.h b/block/blk-mq.h
index dcb2077e4db6..358659fd3175 100644
--- a/block/blk-mq.h
+++ b/block/blk-mq.h
@@ -40,7 +40,7 @@ struct blk_mq_ctx {
 struct request_wrapper {
 	/* Time that I/O was counted in part_get_stat_info(). */
 	u64 stat_time_ns;
-} ____cacheline_aligned_in_smp;
+} ____cacheline_aligned;
 
 static inline struct request_wrapper *request_to_wrapper(void *rq)
 {
-- 
2.33.0

